test/launcher.test.ts

```diff
@@ -70,37 +70,6 @@ describe('Launcher', () => {
                 const mockConf: OsConfig = { urlPath: 'mac/x64', manifestLocation: '', namedPipeName: '', executablePath: '' };
                 await doesntThrowAsync(async () => await download(version, location, mockConf));
             });
-
-            it('parses proxy env variable', () => {
```

Comment:
Deleting tests and not adding any  ?

Reply:
I had already added one in a previous PR that covers this. Deleted the older ones that are not needed anymore.

Reply:
👍